invokeai/frontend/web/src/features/gallery/store/gallerySlice.ts

```diff
@@ -41,6 +41,10 @@ export const gallerySlice = createSlice({
       state.galleryView = 'images';
     },
     autoAddBoardIdChanged: (state, action: PayloadAction<BoardId>) => {
+      if (!action.payload) {
```

Comment:
I couldn't actually even find where this being set to `null` - the initial state has it defined as "none"

Reply:
I just reviewed every instance of `autoAddBoardId`, the action that sets it, and any times it is set directly in the `gallerySlice` reducers. Cannot find anywhere that would result in it being set with `null` or `undefined`. Weird...